crates/mpt/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+edition = "2021"
+name = "strata-mpt"
+version = "0.1.0"
```

Comment:
Can we add a description? I cannot keep count on how many TLAs we have so far... What is MPT?

Reply:
Sure, no problem. It's Merkle Patricia Trie.

Reply:
Oh now it makes total sense. Yeah MPTs of course... But the description is very welcomed.